pkgs/development/tools/steamos-devkit/default.nix

```
@@ -0,0 +1,117 @@
+{ lib
+, fetchFromGitHub
+, fetchFromGitLab
+, writeScript
+, python3
+, makeDesktopItem
+, pkg-config
+, SDL2
+}:
+let
+  # steamos-devkit requires a build of the unreleased pyimgui 2.0 branch, move to pythonPackages when 2.0 is released.
+  pyimgui = python3.pkgs.buildPythonPackage rec {
+    name = "pyimgui";
+    version = "1f095af5886f424ee12f26fa93b108b6420fafa4"; # 2.0 dev
```

```suggestion
    version = "unstable-XXXX-XX-XX";
```
XXXX-XX-XX is the date of the commit